docs/tutorials/dlinktree-builder.md

```diff
@@ -0,0 +1,563 @@
+# Building a Dlinktree maker and Uploading it to the DWeb
+
+## Introduction
+
+This tutorial walks you through creating a Dlinktree builder - a simple web application that allows you to assemble links to your social media profiles, websites, or any other URLs, directly within the code editor developed in our previous tutorial. Once built, you'll be able to upload your Dlinktree to the decentralized web using IPFS or Hypercore protocols and share it with your friends.
+
+## Prerequisites
+
+- Basic knowledge of [HTML](https://developer.mozilla.org/en-US/docs/Learn/HTML), [CSS](https://developer.mozilla.org/en-US/docs/Learn/CSS), and [JavaScript](https://developer.mozilla.org/en-US/docs/Learn/JavaScript).
+- Agregore browser installed on your machine.
+
+## Setting yourself up
+
+1. **Install Agregore Browser:** Download and install the Agregore browser from [Agregore's official website](https://agregore.mauve.moe/).
+2. **Navigate to the P2Pad: code editor:** This tutorial will use the [P2Pad](https://agregore.mauve.moe/docs/examples/p2pad/) web app that was built during a previous tutorial.  
+
+## Step 1: HTML Structure
+
+In this step, we'll lay the foundation of the Dlinktree builder. The HTML structure provides the skeleton of your web application, defining how content is organized and displayed.
+
+First, navigate to the [P2Pad](https://agregore.mauve.moe/docs/examples/p2pad/) code editor in your Agregore browser. In the HTML container, add the following content to the HTML quadrant seen in the top left (or top row if on mobile):
+
+```html
+<html lang="en">
```

Comment:
If this is being put into the code editor would any of these html tags even make sense?

Also how about we add the meta tags and stuff by default into the editor.

Reply:
I see where this might become redundant but I guess these makes sense in the context that we are building the builder itself.
In other words, these are put into the code editor to be uploaded as its own DLink tree. 
The meta tag specifically defines the viewport so that the resulting DLinktree builder is mobile-friendly..
If you think these should be defaults, I could edit the P2Pad itself. 

My argument for keeping them would be that it allows users to edit them and open up possibilities for alternate languages, charset and metatags.

Reply:
I think the mobile friendly thing should be default. Mind double checking what the published HTML looks like? I worry it'd be something like `<html>{headerstuff}<html><body>...app content`

Reply:
Unless I misunderstood where that HTML is being pasted :sweat_smile:

Reply:
You were right. These elements were misplaced. I've removed these from the tutorial. These should indeed be set by default into the editor